pdns/pdns_recursor.cc

```diff
@@ -4499,7 +4499,7 @@ static void setupNODThread()
   }
 }
 
-static void parseNODWhitelist(const std::string& wlist)
+static void parseNODIgnorelist(const std::string& wlist)
```

Comment:
a whitelist and an ignore list seem to be eachother's opposites?

Reply:
Original code used "whitelisted" to not report as newly observed. So this is consitent.